src/pages/login/login.jsx

```diff
@@ -177,7 +177,7 @@ const Login = () => {
                                         buttonSize='md'
                                     />
                                 </button>
-                                <Link href='/signUp/signUp'>
+                                <Link href='/signUp'>
```

Comment:
@Lilicod if we rename the folder to all lower case signup instead of signUp ...  it would be better, what do you think ?